.github/workflows/actions.yml

```diff
@@ -62,6 +62,16 @@ jobs:
           flags: keras.applications,keras.applications-${{ matrix.backend }}
           files: apps-coverage.xml
           fail_ci_if_error: true
+      - name: Test integrations
+        run: |
+          python integration_tests/distribute_training_test.py
+          python integration_tests/import_test.py
+          python integration_tests/numerical_test.py
```

Comment:
We can remove this one, it relies on tf.keras

Reply:
Removed, thanks!